Build fails with GCC 10

## Expected Behavior

Build of the `develop` branch should succeed on Arch Linux with GCC 10.

## Actual Behavior

Build fails with the following message:

```
g++ -c -pipe -D_GLIBCXX_USE_CXX11_ABI=0 -Wno-expansion-to-defined -g -std=gnu++11 -pthread -pthread -pthread -pthread -D_REENTRANT -Wall -Wextra -Wno-unused-result -Wno-unused-parameter -Wno-unused-variable -Wno-switch -Wno-comment -Wno-unused-but-set-variable -Wno-unused-result -Wno-unused-parameter -Wno-unused-variable -Wno-switch -Wno-comment -Wno-unused-but-set-variable -Wno-unused-result -Wno-unused-parameter -Wno-unused-variable -Wno-switch -Wno-comment -Wno-unused-but-set-variable -Wno-unused-result -Wno-unused-parameter -Wno-unused-variable -Wno-switch -Wno-comment -Wno-unused-but-set-variable -Wno-unused-result -Wno-unused-parameter -Wno-unused-variable -Wno-switch -Wno-comment -Wno-unused-but-set-variable -fPIC -DUNICODE -DQT_THREAD_SUPPORT -DQT_CORE_LIB -DQT_GUI_LIB -DQT_MESSAGELOGCONTEXT -DUNIX -DUNIX -DUNIX -DQT_DISABLE_DEPRECATED_BEFORE=0 -DUNIX -DUNIX -DQT_OPENGL_LIB -DQT_WEBENGINEWIDGETS_LIB -DQT_PRINTSUPPORT_LIB -DQT_WIDGETS_LIB -DQT_MULTIMEDIA_LIB -DQT_WEBENGINECORE_LIB -DQT_QUICK_LIB -DQT_GUI_LIB -DQT_XML_LIB -DQT_QMLMODELS_LIB -DQT_WEBCHANNEL_LIB -DQT_QML_LIB -DQT_NETWORK_LIB -DQT_POSITIONING_LIB -DQT_CORE_LIB -I. -Isrc/core -Isrc/shape -Isrc/gui -Isrc/control -Isrc/app -Isrc/core -Isrc/shape -Isrc/gui -Isrc/control -Isrc/app -Isrc/core -Isrc/shape -Isrc/gui -Isrc/control -Isrc/app -Isrc/gui/contrib/qtpropertybrowser/src -Isrc/gui/contrib/qtpropertybrowser-extension -Isrc/core -Isrc/shape -Isrc/gui -Isrc/control -Isrc/app -Isrc/control/qosc -I/home/dburnsii/Documents/mapmap/src/control/qosc/contrib/packosc -Isrc/core -Isrc/shape -Isrc/gui -Isrc/control -Isrc/app -isystem /usr/include/gstreamer-1.0 -isystem /usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -isystem /usr/include/orc-0.4 -isystem /usr/include/qt -isystem /usr/include/qt/QtOpenGL -isystem /usr/include/qt/QtWebEngineWidgets -isystem /usr/include/qt/QtPrintSupport -isystem /usr/include/qt/QtWidgets -isystem /usr/include/qt/QtMultimedia -isystem /usr/include/qt/QtWebEngineCore -isystem /usr/include/qt/QtQuick -isystem /usr/include/qt/QtGui -isystem /usr/include/qt/QtXml -isystem /usr/include/qt/QtQmlModels -isystem /usr/include/qt/QtWebChannel -isystem /usr/include/qt/QtQml -isystem /usr/include/qt/QtNetwork -isystem /usr/include/qt/QtPositioning -isystem /usr/include/qt/QtCore -Isrc/app/mocs -I/usr/lib/qt/mkspecs/linux-g++ -o src/app/objs/MappingItemDelegate.o src/gui/MappingItemDelegate.cpp
src/gui/MappingItemDelegate.cpp: In member function ‘virtual void mmp::MappingItemDelegate::paint(QPainter*, const QStyleOptionViewItem&, const QModelIndex&) const’:
src/gui/MappingItemDelegate.cpp:141:22: error: aggregate ‘QPainterPath mappingSoloPanel’ has incomplete type and cannot be defined
  141 |         QPainterPath mappingSoloPanel;
      |                      ^~~~~~~~~~~~~~~~
src/gui/MappingItemDelegate.cpp:153:22: error: aggregate ‘QPainterPath mappingLockPanel’ has incomplete type and cannot be defined
  153 |         QPainterPath mappingLockPanel;
      |                      ^~~~~~~~~~~~~~~~
make: *** [Makefile:2318: src/app/objs/MappingItemDelegate.o] Error 1
```

## Steps to Reproduce the Problem

  1. Install the latest version of Manjaro with necessary dependencies
  2. Follow steps in the `INSTALL.md` file

## Specifications

  - Version: develop branch (commit `fac1cbd6e7e352772e35816c01f23fc2d6d057a4`)
  - Platform: Manjaro Linux (x64)

--- a/src/gui/MappingItemDelegate.cpp
+++ b/src/gui/MappingItemDelegate.cpp
@@ -18,6 +18,7 @@
  */
 
 #include "MappingItemDelegate.h"
+#include <QPainterPath>
 
 namespace mmp {
 

